driver/src/main/java/org/kaazing/k3po/driver/internal/Robot.java

```diff
@@ -284,23 +287,40 @@ public void exceptionCaught(ChannelHandlerContext ctx, ExceptionEvent e) throws
         return new CompositeChannelFuture<>(channel, bindFutures);
     }
 
-    private void startConfiguration() {
+    private void startConfiguration() throws Exception {
         /* Connect to any clients */
-        for (ClientBootstrap client : configuration.getClientBootstraps()) {
-
-            if (LOGGER.isDebugEnabled()) {
-                LOGGER.debug("[id:           ] connect " + client.getOption("remoteAddress"));
+        for (final ClientBootstrapResolver clientBootstrapResolver : configuration.getClientBootstrapResolvers()) {
```

Comment:
`clientResolver` (not `clientBootstrapResolver`) to match `serverResolver`, right?